src/runner.rs

```diff
@@ -191,13 +194,16 @@ pub mod platform {
     use winreg::enums::*;
 
     pub fn firefox_default_path() -> Option<PathBuf> {
+        if let Some(path) = find_binary("firefox.exe") {
+            return Some(path)
+        }
         let opt_path = firefox_registry_path().unwrap_or(None);
```

Comment:
If people are not expecting Firefox to be picked up from `%path%` at all on Windows, why not remove the condition altogether?